### PR TITLE
Feature/adding seaclouds dc to catalog

### PR DESCRIPTION
**Pull request by MicheleGuerriero**
This PR adds the deployment of the seaclouds-dc in the catalog.bom in order to be deployed within the SeaClouds Platform. At the end of the deployment of the seaclouds-dc it is also installed a monitoring rule that need to be statically added to the Monitor Manager. @kiuby88 @rosogon can you have a look?

### CODE REVIEW

**Comment by andreaturli** [comment]
have you tested this new bom with the installer?

**Comment by kiuby88** [comment]
I have tested this PR. The I have tested the new .bom and it does not break the installation process. But I have found some little issues during the application deployment. I have opened a [PR](https://github.com/MicheleGuerriero/SeaCloudsPlatform/pull/2) to @MicheleGuerriero's branch. Moreover, I think SLA does not works very well, so please @rosogon could you take a look?
All that remains now is the final effort, the last push :)
Thanks a lot.

**Comment by kiuby88** [comment]
@rosogon . SLa produces [this agreement](https://gist.github.com/kiuby88/d5f233d1b25efe25ef3395c1fee40582#file-sla-error-html) it looks that the connection between the sla-core and the sla-db is wrong.

**Comment by kiuby88** [comment]
Thanks @MicheleGuerriero

**Comment by kiuby88** [comment]
@rosogon MySQL avoid the driver problems, but the sla-service is not working correctly. The database can not be accessed, something went wrong about the credentials. Take a look please :) and let me now

**Comment by kiuby88** [comment]
@rosogon , @MicheleGuerriero has just updated the PR. It is ready from my side. Take a look :)

**Comment by kiuby88** [comment]
@rosogon @perezp take a look please

File: byon/files/seaclouds-catalog.bom [hunk 1]
```diff
@@ -409,6 +409,77 @@ brooklyn.catalog:
...
+    name: SeaCloudsDc
+    description: SeaClouds Data Collector 
+    item:
+      type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+      id: seacloudsdc
+      name: SeaClouds Data Collector
+
+      brooklyn.config:
+        runtimeFiles:
+          https://oss.sonatype.org/content/repositories/snapshots/eu/seaclouds-project/monitor/seaclouds-data-collector/1.1.0-SNAPSHOT/seaclouds-data-collector-1.1.0-20160421.085909-4.jar : seaclouds-dc.jar
```
Comment (rosogon) [inline byon/files/seaclouds-catalog.bom]: This url is not available anymore. You should take the same approach as the other components. E.g:  `https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=planner-service&v=LATEST&e=jar`

**Comment by kiuby88** [comment]
@MicheleGuerriero you have avoid my commit and it is mandatory to use the installer so please, included it in your PR.

**Comment by kiuby88** [comment]
Probably, you should do a pull of the remote branch before rebase branch

**Comment by kiuby88** [comment]
@MicheleGuerriero I have opened a PR to your branch again https://github.com/MicheleGuerriero/SeaCloudsPlatform/pull/5
It should solve the installation issues of this PR

**Comment by kiuby88** [comment]
+1 from mi side. 
You can use [this topology](https://gist.githubusercontent.com/rosogon/79a50582233ca3bce5e627ccd438d45e/raw/2ef921ed5226ddd73928b33772710fb942b2755b/1_topology) in order to facilitate the testing

**Comment by perezp** [comment]
The seaclouds-data-collector of this PR is deployed when the platform is installed. I've tested it.
 
The information for this module is created in the DAM when the DAM is generated during the application design. I've seen it. 
So, +1. 

Nevertheless, I had some issues with the deployment of the application I had tested, but I'm quite confident that these issues are beyond the concerns of this PR.

**Pull request merged**